Config option - injecting document.domain (#5974)
cypress-io/cypress-documentation: docs/app/references/configuration.mdx

and does not need to be specified (and can't be overridden). See [`e2e`](#e2e)
or [`component`](#component) testing-specific options.

### injectDocumentDomain

This option is <Badge type="caution">deprecated</Badge>, and will be removed in the next
major version of Cypress.

> Set this configuration option is `true` to instruct Cypress to

Reviewer: ```suggestion
Set this configuration option to `true` to instruct Cypress to
```

Author: addressed in https://github.com/cypress-io/cypress-documentation/pull/5974/commits/644aa612b9a136472c6f1be955f49458f3770b70

[inject document.domain](/app/guides/cross-origin-testing#What-Cypress-does-under-the-hood)
into your test application. This can reduce the need for `cy.origin()` when [navigating
between subdomains](/app/guides/cross-origin-testing), but comes with compatibility
caveats for some sites.

This configuration option is provided to ease the transition between `cy.origin()`'s behavior
in Cypress 13 and the default behavior in Cypress 14. It will be removed in Cypress 15.
[Read the Cypress 14 migration guide](/app/references/migration-guide#Migrating-to-Cypress-140) to understand how to update your tests to remove
the need to set this flag.

This configuration value _must_ be set to true if you are running tests with experimental
WebKit support, as `cy.origin` is not yet supported in WebKit.

:::caution

#### Known Incompatibilities

Setting this configuration value to `true` can cause the application you are testing

> to behave in unexpected ways. This is especially true if a site your tests visit
> sets the [`Origin-Agent-Cluster`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin-Agent-Cluster)

Reviewer: ```suggestion
to behave in unexpected ways. This is especially true if the site under test
sets the [`Origin-Agent-Cluster`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin-Agent-Cluster)
```

Author: addressed in https://github.com/cypress-io/cypress-documentation/pull/5974/commits/644aa612b9a136472c6f1be955f49458f3770b70

and does not need to be specified (and can't be overridden). See [`e2e`](#e2e)
or [`component`](#component) testing-specific options.

### injectDocumentDomain

> This option is <Badge type="caution">deprecated</Badge>, and will be removed in the next
> major version of Cypress.

Reviewer 2: Do we want to be more explicit here?
```suggestion
This option is <Badge type="caution">deprecated</Badge>, and will be removed in Cypress. 15
```

Set this configuration option to `true` to instruct Cypress to
[inject document.domain](/app/guides/cross-origin-testing#What-Cypress-does-under-the-hood)
into your test application. This can reduce the need for `cy.origin()` when [navigating
between subdomains](/app/guides/cross-origin-testing), but comes with compatibility
caveats for some sites.

This configuration option is provided to ease the transition between `cy.origin()`'s behavior
in Cypress 13 and the default behavior in Cypress 14. It will be removed in Cypress 15.
[Read the Cypress 14 migration guide](/app/references/migration-guide#Migrating-to-Cypress-140) to understand how to update your tests to remove
the need to set this flag.

This configuration value _must_ be set to true if you are running tests with experimental
WebKit support, as `cy.origin` is not yet supported in WebKit.

:::caution

#### Known Incompatibilities

Setting this configuration value to `true` can cause the application you are testing

> to behave in unexpected ways. This is especially true if a site your tests visit

Reviewer 2: there are implications if your site sets it, but there are also implications where the browser defaults the `origin-agent-cluster` into buckets, which can cause problems. I think this is also worth mentioning
